app/src/main/java/com/guru/composecookbook/ui/home/dialogs/DialogsActivity.kt

```diff
@@ -284,6 +308,37 @@ fun ShowDialog(type: DialogType, onDismiss: () -> Unit) {
                 onDismissRequest = onDismiss,
                 shape = RoundedCornerShape(16.dp)
             )
+        DialogType.DATEPICKER -> {
+            val context = LocalContext.current
+            (context as? FragmentActivity)?.supportFragmentManager?.let { manager ->
+
+                val builder = MaterialDatePicker.Builder.datePicker()
```

Comment:
I am not sure it's Compose component. Hopefully we will have native compose component for date and time picker.

Reply:
Yes, but I don't found composable for this

Reply:
Still there is no material design's date picker for compose

Reply:
Yup so far no :/

Reply:
Till then we can use this, then we can replace with composable.

Reply:
Yes definitely, thanks for adding :)

Reply:
Is there any material theme adapter that can create theme for AndroidView

Reply:
There is [Theme Adapter](https://material-components.github.io/material-components-android-compose-theme-adapter/) But it's for exporting existing themes in Compose.

Reply:
Yes, but opposite of that.
Like if i want to customise DatePicker or TimePicker we have to define XML styling for those.

Reply:
Yup not that I am aware of yet. But let me dig haha

Reply:
Sure, I will also search for that.